static/js/data_handler.js

```diff
@@ -74,6 +75,46 @@ export let dataHandler = {
     },
     createNewCard: function (cardTitle, boardId, statusId, callback) {
         // creates new card, saves it and calls the callback function with its data
-    }
+    },
     // here comes more features
+    submitNewTitle: function submit_new_title(event) {
+                        let header = event.currentTarget.parentElement;
```

Comment:
and use ctrl+alt+L (auto-format) on all files to make them consistently formatted